### PR TITLE
ast: improve test cover to 70%

### PR DESCRIPTION
**Pull request by tiancaiamao**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: ast/misc_test.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
+package ast_test
```
Comment (zimulala) [inline ast/misc_test.go]: Add license.

Comment (zimulala) [inline ast/misc_test.go]: `ast_test` -> `ast` ? Why use `package ast_test` and `import ast`?

Comment (tiancaiamao) [inline ast/misc_test.go]: cycle dependency:  ast -> parse -> ast

**Comment by zimulala** [comment]
LGTM

**Pull request merged**